photonlib-cpp-examples/apriltagExample/simgui-ds.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
Also maybe gitignore?

Reply:
I vote no, so I can set up WASD bindings for driving around